sdk/ai/azure-ai-agents/samples/agents_tools/sample_agents_mcp.py

```diff
@@ -127,6 +128,7 @@
                 agents_client.runs.submit_tool_outputs(
                     thread_id=thread.id, run_id=run.id, tool_approvals=tool_approvals
                 )
+                run = agents_client.runs.process(thread_id=thread.id, run_id=run.id)
```

Comment:
I think having another process method is probably confusing for customers. They cannot intuitively guess what is needed here.